docs/network-flow-visibility.md

```diff
@@ -171,6 +190,136 @@ through [Antrea Agent apiserver endpoint](prometheus-integration.md):
+
+Flow Aggregator is deployed as a Kubernetes Service. The main functionality of Flow
+Aggregator is to store, correlate and aggregate the flow records received from the
+Flow Exporter of Antrea Agents. More details on the functionality are provided in
+the [Supported Capabilities](#supported-capabilities-1) section.
+
+Flow Aggregator is implemented as IPFIX mediator, which
+consists of IPFIX Collector Process, IPFIX Intermediate Process and IPFIX Exporter
+Process. We use the [go-ipfix](https://github.com/vmware/go-ipfix) library to implement
+the Flow Aggregator.
+
+### Deployment
+
+To deploy a released version of Flow Aggregator Service, pick a deployment manifest from the
+[list of releases](https://github.com/vmware-tanzu/antrea/releases). For any
+given release `<TAG>` (e.g. `v0.12.0`), you can deploy Flow Aggregator as follows:
+
+```bash
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/flow-aggregator.yml
+```
+
+To deploy the latest version of Flow Aggregator Service (built from the master branch), use the
+checked-in [deployment yaml](/build/yamls/flow-aggregator.yml):
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/flow-aggregator.yml
+```
+
+### Configuration
+
+The following configuration parameters have to be provided through the Flow Aggregator
+ConfigMap. `externalFlowCollectorAddr` is a mandatory parameter. We provide an example
+value for this parameter in the following snippet.
+
+```yaml
+flow-aggregator.conf: |
+  # Provide the flow collector address as a string with format <IP>:<port>[:<proto>], where proto is tcp or udp.
+  # If no L4 transport proto is given, we consider tcp as default.
+  externalFlowCollectorAddr: "192.168.86.86:4739:tcp"
+  
+  # Provide flow export interval as a duration string. This determines how often the flow aggregator exports flow
+  # records to the flow collector.
+  # Flow export interval should be greater than or equal to 1s (one second).
+  # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+  #flowExportInterval: 60s
+  
+  # Provide the transport protocol for the flow aggregator collecting process, which is tcp or udp.
+  #aggregatorTransportProtocol: "tcp"
+```
+
+Please note that the default values for `flowExportInterval` and `aggregatorTransportProtocol`
+parameters are set to `60s` and `tcp`, respectively. Please modify them as per your
+requirements.
+
+### IPFIX Information Elements (IEs) in an Aggregated Flow Record
+
+In addition to IPFIX information elements provided in the [above section](#ipfix-information-elements-ies-in-a-flow-record),
+the Flow Aggregator adds the following fields to the flow records.
+
+#### IEs from IANA-assigned IE registry
+
+| IPFIX Information Element  | Enterprise ID | Field ID | Type        |
+|----------------------------|---------------|----------|-------------|
+| originalExporterIPv4Address|      0        |   403    | ipv4Address |
```

Comment:
we also support originalExporterIPv6Address here.

Reply:
Let's add it when we support IPv6?

Reply:
Sure. Just realized even though it is the exporter address, it is still in flow aggregator scope. (sorry for the last minute review :))